src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ListExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
+{
+    internal static class ListExtensions
+    {
+
```

Comment:
nit: extra line

Reply:
just curious about is there any Linting tools performing emtpy line check before I commit code?

Reply:
We dont have any linting tools in this repo yet

Reply:
https://learn.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options?view=vs-2022#supported-settings

Looks like VS natively supports [EditConfig](https://editorconfig.org/) which is popular among other languages. But not sure if it supports rules like extra line around if blocks... Anyway, we might add some linting tools some day, if necessary.  :)